diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/16.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/16.cc
index 54c0934..4b1edfa 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/16.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/16.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2004-03-02  Paolo Carlini  <pcarlini@suse.de>
 
@@ -37,7 +37,7 @@ void test01()
   // basic construction
   locale loc_c = locale::classic();
   locale loc_de = locale("de_DE@euro");
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_de );
   VERIFY( loc_c != loc_hk );
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/18.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/18.cc
index f903a75..8157f91 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/18.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/18.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2004-03-15  Paolo Carlini  <pcarlini@suse.de>
 
@@ -36,7 +36,7 @@ void test01()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
   
   iterator_type end, end01, end02;
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/2.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/2.cc
index c1b8e59..a117f75 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-12 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test02()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // total EPA budget FY 2002
@@ -80,7 +80,7 @@ void test02()
   VERIFY( result11 == digits4 );
   VERIFY( err11 == ios_base::eofbit );
 
-  // for the "en_HK" locale the parsing of the very same input streams must
+  // for the "zh_HK" locale the parsing of the very same input streams must
   // be successful without showbase too, since the symbol field appears in
   // the first positions in the format and the symbol, when present, must be
   // consumed.
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/4.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/4.cc
index f48cc0e..01462ff 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/4.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/4.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-12 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test04()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // input less than frac_digits
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_env.cc
index c0a769d..5a5e096 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_env.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "de_DE" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_locale.cc
index 795271f..660a1d0 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/16.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/16.cc
index 4736617..ea5ebdd 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/16.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/16.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2004-03-02  Paolo Carlini  <pcarlini@suse.de>
 
@@ -37,7 +37,7 @@ void test01()
   // basic construction
   locale loc_c = locale::classic();
   locale loc_de = locale("de_DE@euro");
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_de );
   VERIFY( loc_c != loc_hk );
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/18.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/18.cc
index c1226ad..82e2800 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/18.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/18.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2004-03-15  Paolo Carlini  <pcarlini@suse.de>
 
@@ -36,7 +36,7 @@ void test01()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
   
   iterator_type end, end01, end02;
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/2.cc
index 15bccc6..537e542 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-12 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test02()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // total EPA budget FY 2002
@@ -80,7 +80,7 @@ void test02()
   VERIFY( result11 == digits4 );
   VERIFY( err11 == ios_base::eofbit );
 
-  // for the "en_HK" locale the parsing of the very same input streams must
+  // for the "zh_HK" locale the parsing of the very same input streams must
   // be successful without showbase too, since the symbol field appears in
   // the first positions in the format and the symbol, when present, must be
   // consumed.
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/4.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/4.cc
index bfb803d..e848e4b 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/4.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/4.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-12 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test04()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // input less than frac_digits
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_env.cc
index c0a769d..5a5e096 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_env.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "de_DE" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_locale.cc
index 795271f..660a1d0 100644
--- a/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_get/get/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/char/2.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/char/2.cc
index 6d3136c..31998c6 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-27 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_env.cc
index 3b13bdc..39d94f8 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_env.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "de_DE" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_locale.cc
index b160fb7..779672e 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/2.cc
index 84331bd..20b3fdd 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-27 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_env.cc
index 3b13bdc..39d94f8 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_env.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "de_DE" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_locale.cc
index b160fb7..779672e 100644
--- a/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/money_put/put/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE@euro" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/char/3.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/char/3.cc
index f3e9fc3..bf25a4e 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/char/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/char/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-11-21 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -34,7 +34,7 @@ void test03()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_env.cc
index b8776a3..15939c4 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_locale.cc
index 6f937a9..8e264c9 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/3.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/3.cc
index 387e31e..255c1d5 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-11-21 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -34,7 +34,7 @@ void test03()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_env.cc
index b8776a3..15939c4 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_locale.cc
index 6f937a9..8e264c9 100644
--- a/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_get/get/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/char/3.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/char/3.cc
index d8973fe..35cccc4 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/char/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/char/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-11-19 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -34,7 +34,7 @@ void test03()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_env.cc
index 8ed8175..7885d8d 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_locale.cc
index 867992f..79a4429 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/3.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/3.cc
index 47ed26c..92eed33 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-11-19 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -34,7 +34,7 @@ void test03()
 
   // basic construction
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_c != loc_hk );
 
   // sanity check the data is correct.
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_env.cc
index bb6bec8..0cca16e 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_locale.cc
index d85c72a..e7a98e9 100644
--- a/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/num_put/put/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/2.cc
index d0e5ca4..a7f546f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -76,7 +76,7 @@ void test02()
   VERIFY( time10.tm_year == time_bday.tm_year );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str("Sunday, April 04, 1971"); 
   iterator_type is_it20(iss);
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/2.cc
index bb6d72b..76c3bbd 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -76,7 +76,7 @@ void test02()
   VERIFY( time10.tm_year == time_bday.tm_year );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str(L"Sunday, April 04, 1971"); 
   iterator_type is_it20(iss);
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_date/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/2.cc
index dc117a3..b38efb0 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -35,7 +35,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -63,7 +63,7 @@ void test02()
   VERIFY( time10.tm_mon == time_bday.tm_mon );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   const time_get<char>& tim_get2 = use_facet<time_get<char> >(iss.getloc()); 
   iss.str("April"); 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/2.cc
index 89ed391..7ce7510 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -64,7 +64,7 @@ void test02()
   VERIFY( time10.tm_mon == time_bday.tm_mon );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   const time_get<wchar_t>& tim_get2 = use_facet<time_get<wchar_t> >(iss.getloc()); 
   iss.str(L"April"); 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_monthname/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/2.cc
index 0df5322..ac9dfe7 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test02()
 
   // basic construction and sanity check
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   const string empty;
@@ -48,7 +48,7 @@ void test02()
   const ios_base::iostate good = ios_base::goodbit;
   ios_base::iostate errorstate = good;
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str("12:00:00 PST"); 
   // Hong Kong in California! Well, they have Paris in Vegas... this
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/2.cc
index 7294661..ac088fd 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -35,7 +35,7 @@ void test02()
 
   // basic construction and sanity check
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   const wstring empty;
@@ -48,7 +48,7 @@ void test02()
   const ios_base::iostate good = ios_base::goodbit;
   ios_base::iostate errorstate = good;
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str(L"12:00:00 PST"); 
   // Hong Kong in California! Well, they have Paris in Vegas... this
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_time/wchar_t/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/2.cc
index 1b11369..83f0eb9 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -65,7 +65,7 @@ void test02()
   VERIFY( time10.tm_wday == time_bday.tm_wday );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str("Sunday"); 
   iterator_type is_it20(iss);
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_env.cc
index 8f11671..5cdfd8f 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_locale.cc
index a4edc45..fea106b 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/char/wrapped_locale.cc
@@ -1,6 +1,6 @@
 // { dg-require-namedlocale "ja_JP.eucjp" }
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/2.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/2.cc
index dbab614..aecf932 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/2.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/2.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-09-21 Benjamin Kosnik  <bkoz@redhat.com>
@@ -36,7 +36,7 @@ void test02()
 
   // basic construction and sanity checks.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   locale loc_de = locale("de_DE");
   VERIFY( loc_hk != loc_c );
   VERIFY( loc_hk != loc_de );
@@ -65,7 +65,7 @@ void test02()
   VERIFY( time10.tm_wday == time_bday.tm_wday );
   VERIFY( errorstate == ios_base::eofbit );
 
-  // inspection of named locales, en_HK
+  // inspection of named locales, zh_HK
   iss.imbue(loc_hk);
   iss.str(L"Sunday"); 
   iterator_type is_it20(iss);
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_env.cc
index bcd8f51..c748e00 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_env.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 
 // 2001-08-15 Benjamin Kosnik  <bkoz@redhat.com>
diff --git a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_locale.cc
index 610c7db..d22d185 100644
--- a/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_get/get_weekday/wchar_t/wrapped_locale.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "de_DE" }
 // { dg-require-namedlocale "ja_JP.eucjp" }
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/char/3.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/char/3.cc
index d4f04c2..ae4070e 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/char/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/char/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-17 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -38,7 +38,7 @@ void test03()
 
   // basic construction and sanity check.
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   // create an ostream-derived object, cache the time_put facet
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/char/7.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/char/7.cc
index 595fadf..bfb2c24 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/char/7.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/char/7.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-17 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -41,7 +41,7 @@ void test07()
 
   // basic construction and sanity check
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   // create an ostream-derived object, cache the time_put facet
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_env.cc
index 495dcda..b5406be 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 // { dg-require-namedlocale "fr_FR@euro" }
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_locale.cc
index 4128ef4..04bf2a0 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/char/wrapped_locale.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 // { dg-require-namedlocale "fr_FR@euro" }
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/3.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/3.cc
index 732b42a..ae78a67 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/3.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/3.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-17 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -38,7 +38,7 @@ void test03()
 
   // basic construction and sanity check
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   // create an ostream-derived object, cache the time_put facet
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/7.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/7.cc
index 7f08c1f..3af3b73 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/7.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/7.cc
@@ -1,4 +1,4 @@
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 
 // 2001-09-17 Benjamin Kosnik  <bkoz@redhat.com>
 
@@ -41,7 +41,7 @@ void test07()
 
   // basic construction and sanity check
   locale loc_c = locale::classic();
-  locale loc_hk = locale("en_HK");
+  locale loc_hk = locale("zh_HK");
   VERIFY( loc_hk != loc_c );
 
   // create an ostream-derived object, cache the time_put facet
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_env.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_env.cc
index 495dcda..b5406be 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_env.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_env.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 // { dg-require-namedlocale "fr_FR@euro" }
 
diff --git a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_locale.cc b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_locale.cc
index a998999..cec9eab 100644
--- a/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_locale.cc
+++ b/libstdc++-v3/testsuite/22_locale/time_put/put/wchar_t/wrapped_locale.cc
@@ -1,5 +1,5 @@
 // { dg-require-namedlocale "de_DE" }
-// { dg-require-namedlocale "en_HK" }
+// { dg-require-namedlocale "zh_HK" }
 // { dg-require-namedlocale "es_ES" }
 // { dg-require-namedlocale "fr_FR@euro" }
 // { dg-require-namedlocale "ja_JP.eucjp" }
