diff --git a/gcc/testsuite/g++.dg/cdce3.C b/gcc/testsuite/g++.dg/cdce3.C
index e6ca58c..b6cc4bf 100644
--- a/gcc/testsuite/g++.dg/cdce3.C
+++ b/gcc/testsuite/g++.dg/cdce3.C
@@ -1,4 +1,4 @@
-/* { dg-do run } */
+/* { dg-do run { target { ! "*-*-freebsd* *-*-dragonfly* *-*-netbsd*" } } } */
 /* { dg-require-effective-target c99_runtime } */
 /* { dg-skip-if "exp2* missing despite C99 runtime" { alpha*-dec-osf5* } } */
 /* { dg-options "-O2 -fmath-errno -fdump-tree-cdce-details  -DGNU_EXTENSION -DLARGE_LONG_DOUBLE -lm" { target { pow10 && large_long_double } } } */
diff --git a/gcc/testsuite/g++.dg/uninit-pred-3_b.C b/gcc/testsuite/g++.dg/uninit-pred-3_b.C
index cfe2113..a66e04b 100644
--- a/gcc/testsuite/g++.dg/uninit-pred-3_b.C
+++ b/gcc/testsuite/g++.dg/uninit-pred-3_b.C
@@ -65,7 +65,7 @@ class M {
 
   void P (int64 t)
     {
-      int cc; /* { dg-excess-errors "note: 'cc' was declared here" } */
+      int cc;
       if (!GetC (&cc))
         return;
 
diff --git a/gcc/testsuite/gcc.dg/builtins-config.h b/gcc/testsuite/gcc.dg/builtins-config.h
index 49ec0dc..b3ab1f3 100644
--- a/gcc/testsuite/gcc.dg/builtins-config.h
+++ b/gcc/testsuite/gcc.dg/builtins-config.h
@@ -13,8 +13,14 @@
 /* PA HP-UX doesn't have the entire C99 runtime.  */
 #elif defined(__AVR__)
 /* AVR doesn't have the entire C99 runtime.  */
-#elif defined(__FreeBSD__) && (__FreeBSD__ < 9)
-/* FreeBSD up to version 8 lacks support for cexp and friends.  */
+#elif defined(__FreeBSD__) && (__FreeBSD__ < 10)
+/* FreeBSD up to version 9 lacks support for cexp and friends.  */
+#elif defined(__DragonFly__)
+/* As of DragonFly 2.13, Full C99 support is not available.
+   It has the same libm functionality as FreeBSD, which is missing
+   several long-double functions. */
+#elif defined(__NetBSD__)
+/* As of NetBSD 5.99, libm is nowhere close to supporting C99 */
 #elif defined(__netware__)
 /* NetWare doesn't have the entire C99 runtime.  */
 #elif defined(__vxworks)
